incubator/hnc/pkg/reconcilers/hierarchical_namespace.go

```diff
@@ -41,6 +45,22 @@ func (r *HierarchicalNamespaceReconciler) Reconcile(req ctrl.Request) (ctrl.Resu
 	log := r.Log.WithValues("trigger", req.NamespacedName)
 	log.Info("Reconciling HNS")
 
+	// Get the names of the current namespace and the hierarchical namespace.
+	nm := req.Name
+	pnm := req.Namespace
+
+	// Set RequiredChildOf (Owner) in the forest of the hierarchical namespace to the
+	// current namespace.
+	// TODO rename RequiredChildOf to Owner in the forest. See issue:
+	//  https://github.com/kubernetes-sigs/multi-tenancy/issues/469
+	r.forest.Get(nm).RequiredChildOf = pnm
```

Comment:
Need TODO to handle conflicts.

I _think_ you can just say `#469` in the code and it will be displayed as a link in GH - but I'm not certain.

Reply:
The TODO is added in the `Reconcile()` before `return`. I tried `#469` and it didn't turn into a link, so I changed it back to a full github link.

Reply:
sg